ICSSoft.STORMNET.DataObject/DataObjectExtension.cs

```diff
@@ -1,13 +1,40 @@
 ﻿namespace ICSSoft.STORMNET
 {
     using System;
+    using System.Linq;
     using System.Linq.Expressions;
 
     /// <summary>
     /// Класс с linq-расширениями для DataObject и его потомков.
     /// </summary>
     public static class DataObjectExtension
     {
+        /// <summary>
+        /// Проверить, установлено ли значение в указанное свойство
+        /// (требуется в случае, когда состояние загрузки -- LightLoaded).
+        /// </summary>
+        /// <typeparam name="T">Тип объекта. Указывать явно не требуется.</typeparam>
+        /// <param name="dataObject">Сам объект, над которым будем проводить проверку.</param>
+        /// <param name="propertyExpression">Выражение, которое будет возвращать поле объекта. Например, "x => x.Свойство1".</param>
+        /// <returns><c>True</c>, если значение инициировано в копии данных.</returns>
+        public static bool CheckLoadedProperty<T>(this T dataObject, Expression<Func<T, object>> propertyExpression) where T : DataObject
```

Comment:
Для чего этот DataObjectExtension вообще нужен? Почему нельзя все эти методы впилить прямо в DataObject?

Reply:
Если впилить прямо в класс, то нельзя использовать короткий синтаксис:
client.CheckLoadedProperty(x => x.Прописка)
Только полный вариант:
client.CheckLoadedProperty<Клиент>(x => x.Прописка)
Иначе ошибки
CS0411: The type arguments for method 'DataObject.CheckLoadedProperties<T>(params Expression<Func<T, object>>[])' cannot be inferred from the usage. Try specifying the type arguments explicitly.
CS1660: Cannot convert lambda expression to type 'string' because it is not a delegate type